### DISCUSSION TITLE
Reverse Proxy fails to open Faikin Main Control Page

### DISCUSSION THREAD

**Discussion by bjappler** [revk/xESP32-Faikout#575] [Q&A]
Hi,

I tried to open the faikin control via a reverse proxy, to control Faikin without VPN outside my home network directly.
Unfortunately the faikin page will not be loaded completely as you can see in the attached screenshot.
I can open open all setting pages by clickin on Settings. Only the "Home" page with the control section will not be loaded correctly.
Can anybody help me? 
<img width="477" alt="faikin" src="https://github.com/user-attachments/assets/7c1a404f-6225-4942-bb75-8b9cd297e710" />

**Comment by revk** [comment]
Does the browser debug show any errors?

**Reply by bjappler** [reply]
here are the error messages
<img width="1292" alt="faikin2" src="https://github.com/user-attachments/assets/6867291d-240c-420a-ad7b-ede7bbf250ca" />

**Reply by revk** [reply]
Someone else said they seemed to have an issue with `wss:` access using the reverse proxy - I don't know enough about it, but if it does not proxy web sockets it is not going to work, sorry.

**Reply by bjappler** [reply]
web sockets was the key!
I activated websocket in my reverse proxy rules as shown here: [https://mlohr.com/websockets-for-synology-dsm/](url) and now it works perfectly.

Thank you very much.

**Reply by revk** [reply]
Thank you, good to know.
